frontend/src/bundles/drag-and-drop/components/drop-container/drop-container.tsx

```diff
@@ -0,0 +1,24 @@
+import { type UniqueIdentifier, useDroppable } from '@dnd-kit/core';
+
+type Properties = {
+    id: UniqueIdentifier;
+    children: React.ReactNode;
+    className?: string;
+};
+
+const DropContainer: React.FC<Properties> = ({ id, children, className }) => {
```

Comment:
: Properties

Reply:
Doesn't change anything because Properties are in generic